src/main/java/org/jitsi/videobridge/cc/AdaptiveTrackProjection.java

```diff
@@ -184,8 +184,12 @@ public boolean accept(@NotNull RawPacket rtpPacket)
         // suspended so that it can raise the needsKeyframe flag and also allow
         // it to compute a sequence number delta when the target becomes > -1.
 
+        RTPEncodingDesc encoding = getSource()
+            .getMediaStreamTrackReceiver().findRTPEncodingDesc(rtpPacket);
```

Comment:
should we handle an encoding not being found? (maybe signaling hasn't come through yet?)

Reply:
yeah, I'll make this more graceful.